style.css

```diff
@@ -88,3 +97,291 @@
 .social-icons-set img {
   margin: 0 9px;
 }
+
+.works-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+  gap: 80px;
```

Comment:
- [ ] Please note that when I open the website in a mid-screen (eg 500+px), your cards are not centred properly 🔽  please try to fix that, you use the `align-items` property to fix the problem.

![2023-03-24_231028](https://user-images.githubusercontent.com/104319462/227659767-3c065f46-5e27-4469-9107-e9232a145d94.png)